src/core/Button/ButtonFilled/styles.ts

```diff
@@ -5,6 +5,7 @@ const useStyles = makeStyles((theme: Theme) => ({
   root: {
     height: "fit-content",
     minWidth: "6.25rem",
+    minHeight: "2.75rem",
```

Comment:
Why is maxHeight required...shouldn't it depend on the font side and the padding (Top, bottom) how tall the button can be.